### DISCUSSION TITLE
tooken是红色括号这段吗？

### DISCUSSION THREAD

**Discussion by 736283428** [0-8-4/miui-auto-tasks#243] [综合讨论｜General]
![Screenshot_2023-12-12-13-19-25-566-edit_com.guoshi.httpcanary.jpg](https://github.com/0-8-4/miui-auto-tasks/assets/30228748/707ec923-cdeb-4d8a-9402-320c635cf98f)

**Comment by Night-stars-1** [comment]
不是的，这个是cookies用的token和签到的token不一样
